README.md

```diff
@@ -91,7 +91,7 @@ Whenever you want to contribute to any project, it is a good practice to make a
 2. Login with your Discord credentials. You will see the following screen (image below and for you, it will probably be a blank one)
     ![homepage](https://raw.githubusercontent.com/commclassroom/classroom-monitor-bot/main/doc/images/HomePage.jpg)
 3. Now click the `New Application` button (refer to the image above)
-4. Name your bot (it can be `Community Classroom bot` or anything else) and press `create`
+4. Name your bot (it can be `WeMakeDevs bot` or anything else) and press `create`
 5. You will be at the Developers Portal where you can customize your bot (refer to the image below)
     ![dashboard](https://raw.githubusercontent.com/commclassroom/classroom-monitor-bot/main/doc/images/Dashboard.jpg)

```

Comment:
These kinds of links should be updated as well.

Reply:
now its okay??